tests/quickchecks.rs

```diff
@@ -1,9 +1,7 @@
-// Miri is very slow. Even with QUICKCHECK_MAX_TESTS=1 the tests in this module run for more than
-// an hour.
```

Comment:
Actually environment variables are invisible to miri, unless you disable its isolation or pass environment variable at compile time. Properly enabling `QUICKCHECK_TESTS=1` actually increases `cargo miri test --all-features` time from ~1:40 to only ~4:00, so i enabled one quickcheck run on miri by default.